src/System.Private.CoreLib/shared/System/Threading/Mutex.Windows.cs

```diff
@@ -42,11 +27,11 @@ private void CreateMutexCore(bool initiallyOwned, string name, out bool createdN
             if (mutexHandle.IsInvalid)
             {
                 mutexHandle.SetHandleAsInvalid();
-#if PLATFORM_UNIX
```

Comment:
It may be a good idea to keep the `PLATFORM_UNIX` ifdefs around these for clarity given that we do not ever expect to take this path on Windows.